custom/layouts/pages/search.ejs

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <%- include('partials/head', {title: template('search.results.title', locals.q)}) %>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" 
+        integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" 
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <body>
+    <%- include('partials/header', {parentLinks: [], title: template('search.results.title', locals.q)}) %>
+
+    <%- include('partials/nav') %>
+
+    <div class="g-body">
+      <div class="g-main-content" id="g-search-page">
+        <% if (locals.results && results.length) { %>
+          <h3 class="search-kicker"><%- template('search.results.kicker', locals.q) %></h3>
+          <% results.forEach((res) => { %>
+          <div class="search-result">
+            <h3><a href="<%= res.path %>"><%= res.prettyName %></a></h3>
+            <%- include('partials/fileType', { mimeType: res.mimeType }) %>
+            <%- template('search.results.') %>
```

Comment:
what's this line for?

Reply:
Whoops.  Thanks for noticing.  That shouldn't be there.